client/mic.py

```diff
@@ -32,18 +33,14 @@ def __init__(self, input_device, output_device,
         self._input_channels = 1
         self._input_bits = 16
         self._input_chunksize = 1024
-        self._threshold = self._get_threshold()
-
-    # Input methods
-    def _get_threshold(self):
-        self._logger.debug("Recording silence...")
-        recording = self._input_device.record(self._input_chunksize,
-                                              self._input_bits,
-                                              self._input_channels,
-                                              self._input_rate)
-        rms = max([audioop.rms(recording.next(), 1) for i in range(30)])
-        self._logger.debug("Finished recording silence, Threshold is %r", rms)
-        return rms
+        self._threshold = 2.0**16
```

Comment:
Doesn't really matter; just needs to be a big number. Happy to change it though if you prefer.